ecosystem/web-wallet/public/inpage.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+const extensionId = 'jmmhmakollmjgkjgnbeppkccgddmjnib'
+
+class Web3 {
```

Comment:
Is it possible to use TS for this file?

Reply:
I mentioned this in the summary, it's complicated to inject a ts file. I'm going to followup at some point to try and move it over but I'll have to do some webpack things